package.json

```diff
@@ -21,9 +21,10 @@
     "test:unit:coverage": "karma start --single-run --reporters spec,coverage",
     "test:functional": "rollup -c --environment BASE_CODE,NPM_PACKAGE_LOCAL && EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe chrome",
     "test:functional:watch": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" ./scripts/watchFunctionalTests.js",
-    "test:functional:ci:chrome": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe 'browserstack:chrome@87:Windows 10'",
-    "test:functional:ci:ie": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe 'browserstack:ie@11:Windows 10'",
-    "test:functional:cron:prod": "ALLOY_ENV=\"prod\" testcafe 'browserstack:chrome@87:OS X'",
+    "test:functional:build": "rollup -c --environment BASE_CODE,NPM_PACKAGE_LOCAL",
```

Comment:
I think a better name for this would be "test:functional:ci:build" as you don't need to call this to run "test:functional" or "test:functional:watch".